_includes/footer.html

```diff
@@ -9,29 +9,29 @@ <h3>{% t navigation.resources %}</h3>
                                 <li><a class="white" href="{{ site.baseurl }}/resources/user-guides/">{% t titles.userguides %}</a></li>
                                 <li><a class="white" href="{{ site.baseurl }}/library/">{% t titles.library %}</a></li>
-                                <li><a class="white" href="{{ site.baseurl_root }}/feed.xml">{% t footer.feed %}</a></li>
+                                <li><a class="white" href="{{ site.baseurl }}/downloads/">{% t titles.downloads %}</a></li>
+                                
                             </ul>
                      </div>
                      <div class="col-sm-3 col-xs-6">
-                        <h3>{% t titles.channels %}</h3>
+                        <h3>{% t footer.reachout %}</h3>
                             <ul class="list-unstyled">
-                                 <li><a class="white" href="irc://chat.freenode.net/#monero">monero</a></li>
-                                 <li><a class="white" href="irc://chat.freenode.net/#monero-dev">monero-dev</a></li>
-                                 <li><a class="white" href="irc://chat.freenode.net/#monero-markets">monero-markets</a></li>
-                                 <li><a class="white" href="irc://chat.freenode.net/#monero-pools">monero-pools</a></li>
-                                 <li><a class="white" href="irc://chat.freenode.net/#monero-community">monero-community</a></li>
-                                 <li><a class="white" href="irc://chat.freenode.net/#monero-translations">monero-translations</a></li>
-                                 <li><a class="white" href="irc://chat.freenode.net/#monero-hardware">monero-hardware</a></li>
+                                 <li><a class="white" href="{{ site.baseurl}}/community/team/">{% t titles.workgroups %}</a></li>
+                                 <li><a class="white" href="{{ site.baseurl}}/community/hangouts/">{% t titles.hangouts %}</a></li>
+                                 <li><a class="white" href="https://lists.getmonero.org/postorius/lists/monero-announce.lists.getmonero.org/">{% t footer.ml %}</a></li>
+                                 <li><a class="white" href="{{ site.baseurl_root }}/feed.xml">{% t footer.feed %}</a></li>
+                                 <li><a class="white" href="https://mattermost.getmonero.org/">Mattermost</a></li>
+                                 <li><a class="white" href="https://telegram.me/monero">Telegram chat</a></li>
+
                             </ul>
                     </div>
                     <div class="col-sm-3 col-xs-6">
                         <h3>{% t navigation.community %}</h3>
                             <ul class="list-unstyled">
-                                 <li><a class="white" href="https://reddit.com/r/monero">Reddit</a></li>
+                                 <li><a class="white" href="{{ site.baseurl}}/community/merchants/">{% t titles.merchants %}</a></li>
```

Comment:
Why remove Reddit link?

Reply:
I added the Hangouts page in the Reach out colum and that includes all platforms where the community is present, reddit included.